packages/backend/src/workers/action.ts

```diff
@@ -143,6 +143,19 @@ worker.on('failed', async (job, err) => {
   }
 })
 
+worker.on('error', (err) => {
+  // catch-all just in case any errors bubble up and potentially crash the worker task
+  logger.error(`Worker errored with ${err.message}`, {
+    err,
```

Comment:
can we exclude `err` for now, in case we run into yesterday problem (err obj too big to log)